src/cascadia/TerminalApp/Pane.cpp

```diff
@@ -446,10 +475,13 @@ void Pane::UpdateFocus()
                                     _control.FocusState() != FocusState::Unfocused;
 
         _lastFocused = controlFocused;
+
+        _border.BorderBrush(_lastFocused ? s_focusedBorderBrush : nullptr);

```

Comment:
qq: should this be when the _pane_ is focused, or should it be tied to the control hosted inside the pane? Each UIElement/Control has a focus event that we could subscribe to, and the pane could handle updating its status when the control gains/loses focus

or will that be bad when we finally decouple active and focused?

Reply:
IMO: 
"when the pane is focused" == "active"
"when the control is focused" == "focused"

So I think the border should be there when the pane is _active_. We don't want the border to disappear when another UI element appears (e.g. the command palette). Also consider a pane where there are many (non-terminal) controls, any of which could be focused.

Ideally, all this will get resolved when we loop back on the active vs focused issue #1205

Reply:
I think it makes more sense for this to be on the pane than the control. Two scenarios come to mind:
- if we support having something other than a TermControl, we can still apply the border
- if we wanted to allow multiple panes to be "selected" for some reason, it still works